new/PRS13900WaldaD.xml

```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13900WaldaD" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Walda Dāwit</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2022-09-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ወልደ፡ ዳዊት፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walda Dāwit</persName>
+                    <floruit notBefore="1743" notAfter="1767"/>
```

Comment:
I am probably asking something trivial: notBefore and notAfter means that Walda Dāwit was certainly born after 1743 and died before 1767. Is this what you meant? Or that he was certainly in existence during the 1743-1767 time frame?

Reply:
I mean that he is attested in a manuscript dated to this period...So that he was certainly in existence at one point in this period, but not necessarily during the entire period.
You are right, it is a bit confusing.
The TEI guidelines for floruit have "contains information about a person's period of activity", the BM guidelines have "contains the date of the major active years of the record's subject". 
I believe that my use is the common one in Bm, but that's not a reason not to improve it - what would you suggest?

Reply:
Probably the best thing to do is to keep the current BM definition of floruit and stick to it. Since 1743-1767 are Walda Dāwit's "major active years" (unless further info comes to light), notBefore and notAfter are fine.